pkg/compactor/bucket_compactor.go

```diff
@@ -773,6 +790,22 @@ func NewBucketCompactorMetrics(blocksMarkedForDeletion prometheus.Counter, reg p
 			Help:    "Difference between now and the max time of a block being compacted in seconds.",
 			Buckets: prometheus.LinearBuckets(86400, 43200, 8), // 1 to 5 days, in 12 hour intervals
 		}),
+		blockUploadsStarted: promauto.With(reg).NewCounter(prometheus.CounterOpts{
+			Name: "cortex_compactor_block_uploads_started_total",
+			Help: "Total number of block uploads started.",
+		}),
+		blockUploadsFailed: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_compactor_block_uploads_failed_total",
+			Help: "Total number of block uploads failed.",
+		}, []string{"reason"}),
+		blockUploadsDuration: promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
+			Name:                            "cortex_compactor_block_upload_duration_seconds",
+			Help:                            "Duration of successful block uploads.",
+			Buckets:                         prometheus.ExponentialBuckets(0.004, 2, 14), // 4ms to 32768ms
```

Comment:
3.2s might be a bit tight for slow object stores and large objects. Maybe bump it to 12.8
```suggestion
			Buckets:                         prometheus.ExponentialBuckets(0.004, 2, 16), // 4ms to 32768ms
```

Reply:
This is 32s. 0.004s = 4ms; 4ms * 2^13 = 32.768s.

Reply:
ok, then the comment is wrong

Reply:
oh, it isn't. i kept reading it and kept seeing 4 digits. I swear i read the thing like 5 times